public/index.html

```diff
@@ -12,8 +17,8 @@
   </head>
 
   <body class="govuk-template__body">
-    <header class="govuk-header" role="banner" data-module="govuk-header">
-      <div class="govuk-header__container govuk-width-container" style="border-bottom: 10px solid #ffffff; margin-left: 20px;">
+    <header class="govuk-header" role="banner" style="border-bottom: 10px solid #0b0c0c" data-module="govuk-header">
```

Comment:
Would it be possible to move these inline styles into a separate file? I don't know how easy it would be to set up compilation for a Sass file (if it doesn't exist already) but a CSS file should do if that's too complicated.